nequip/model/builder_utils.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Union
+
+import torch
+
+from nequip.utils import Config
+from nequip.data import AtomicDataset, AtomicDataDict
+
+
+def add_avg_num_neighbors(
+    config: Config,
+    initialize: bool,
+    dataset: Optional[AtomicDataset] = None,
+    default: Optional[Union[str, float]] = "auto",
```

Comment:
when will the `default` argument be changed? why setting it as an argument? We don't use instantiate on this function, right?